web/src/components/troubleshoot/GenerateSupportBundle.jsx

```diff
@@ -23,12 +24,10 @@ class GenerateSupportBundle extends React.Component {
       totalBundles: null,
       showRunCommand: false,
       isGeneratingBundle: false,
-      initialLoading: true,
```

Comment:
not necessary anymore, we will only depending on isGeneratingBundle to render the button `Analyze {appTitle} `
User should not be able to redirect to `/troubleshoot/generate` if a support bundle is pending.